docs/tutorials/10_effective_dimension.ipynb

```diff
@@ -0,0 +1,763 @@
+      "text/plain": [
+       "<Figure size 447.797x204.68 with 1 Axes>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "num_qubits = 3\n",
+    "# create a feature map\n",
+    "feature_map = ZFeatureMap(feature_dimension=num_qubits, reps=1)\n",
+    "# create a variational circuit\n",
+    "ansatz = RealAmplitudes(num_qubits, reps=1)\n",
+    "\n",
+    "# create quantum circuit\n",
```

Comment:
Rather than this more abstract comment, would it be better to sat combine the feature map and ansatz into a in single circuit.

Now this looks like a `two layer qnn`. Is there some rationale why we chose to do that as a CircuitQNN here rather than use the that. If so would this benefit from some explanation here. I assume this can work fine with a TwoLayerQNN.

Reply:
Yes, this is basically a TwoLayerQNN, and the only reason why it's shown with a Circuit QNN is to have both examples. I can add a remark pointing this out.